### ISSUE TITLE
Question about temporal conv training

### ISSUE THREAD

**Issue by lsrock1** [mpc001/end-to-end-lipreading#11]
Thank you for your code.
I have a question about training.
In temporal conv step, did you use 0.0003 learning rate, no weight decay and adjust lr(scheduler) in this code?
because only in 11 epochs it overfitted and accuracy is too low.

**Comment by mpc001** [comment]
Hi, @lsrock1,

Thanks for your interests! Yes. For the temporalConv training, we got the training accuracy of 41% at the 1st epoch and the training accuracy of 80% at the 11th epoch. Did you train on the pytorch0.3.1?

Kind regards,
Pingchuan Ma

**Comment by lsrock1** [comment]
Thanks for your kind answer!
I trained on the pytorch 0.4.1!
Then, How much epochs did you train for temporal conv?

**Comment by mpc001** [comment]
Hi, @lsrock1,

I think there are some changes between pytorch0.3.1 and pytorch0.4.1.
For me, I took 30 epochs to train the temporalConv.

**Comment by lsrock1** [comment]
Thanks!
I will train again

**Comment by lsrock1** [comment]
I successfully reproduced your accuracy in first stage.
Thank you!
